apps/www/pages/_ssr/front-preview/[...id].tsx

```diff
@@ -0,0 +1,53 @@
+import { GetServerSideProps, NextPage } from 'next'
```

Comment:
AFAICS we don't need to accept multiple path segments here and `id` seems like a front id but is then mapped to `extractId`. Also preview is confusing with the `vorschau` path that we also support.

```diff
-apps/www/pages/_ssr/front-preview/[...id].tsx
+apps/www/pages/_ssr/front/[extractId].tsx
```